### PR TITLE
Change Video.PodID to a string, as per the spec

### PR DESCRIPTION
**Pull request by julian-burger-ttd**

### CODE REVIEW

**Comment by julian-burger-ttd** [comment]
@bsardo Can you please take a look at this?

**Comment by julian-burger-ttd** [comment]
Or perhaps @SyntaxNode ?

**Comment by SyntaxNode** [comment]
Good catch. Thank you for correcting the mistake. Could you please fix the same in the openrtb2 audio struct?

**Comment by julian-burger-ttd** [comment]
Done.

File: adcom1/audio_placement.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ type AudioPlacement struct {
 	//   video ad pod. If multiple impression opportunities within a bid request
 	//   share the same podid, this indicates that those impression opportunities
 	//   belong to the same video ad pod.
-	PodID int64 `json:"podid,omitempty"`
+	PodID string `json:"podid,omitempty"`
```
Comment (SyntaxNode) [inline adcom1/audio_placement.go]: The AdCOM PodIDs are integers, per: https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md

Looks like the ad podding changes to AdCOM don't match the changes to OpenRTB 2.6. I don't closely follow AdCOM, but I'll track down what happened. Let's keep the adcom package int64 to match the spec, but openrtb2 should use a string. I think I see where my original gaffe came from. :)

Comment (julian-burger-ttd) [reply adcom1/audio_placement.go]: Got it. Reverted that change.

**Review by SyntaxNode — APPROVED** [review]

**Review by bsardo — APPROVED** [review]
LGTM

**Pull request merged**